scripts/linux/generic-rust/build.sh

```diff
@@ -101,16 +106,17 @@ if [ "${REDUCED_LINKER}" = '1' ]; then
     # We don't want to disable these for everyone else, so only do it in this script
     # that the CI uses.
     >> "${PROJECT_ROOT}/Cargo.toml" cat <<-EOF
-
 [profile.dev]
 codegen-units = 1
 incremental = false
-
 [profile.test]
 codegen-units = 1
 incremental = false
 EOF
 fi
 
 cd "${PROJECT_ROOT}"
-$CARGO build ${PACKAGES_FORMATTED} ${CARGO_ARGS} --target "$TARGET"
+BUILD_COMMAND="$CARGO build ${PACKAGES_FORMATTED} $(echo ${CARGO_ARGS}) --target \"$TARGET\""
```

Comment:
```suggestion
BUILD_COMMAND="$CARGO build ${PACKAGES_FORMATTED} ${CARGO_ARGS} --target \"$TARGET\""
```
this is also works

Reply:
I had a misconception and thought this fixed what your error was originally. Thanks for cleaning it up. I would have never realized.